scalanativelib/src/ScalaNativeModule.scala

```diff
@@ -84,7 +84,13 @@ trait ScalaNativeModule extends ScalaModule { outer =>
   }
 
   override def scalaLibraryIvyDeps = T {
-    if (isScala3(scalaVersion())) Agg.empty[Dep] else super.scalaLibraryIvyDeps()
+    super.scalaLibraryIvyDeps().map(dep =>
+      dep.copy(cross = dep.cross match {
+        case c: CrossVersion.Constant => c.copy(platformed = false)
+        case c: CrossVersion.Binary => c.copy(platformed = false)
+        case c: CrossVersion.Full => c.copy(platformed = false)
+      })
+    )
```

Comment:
Looks like the match on the concrete `CrossType` is superfluous, as we always set `platformed = false`, for each of the possible sealed trait instances.

Reply:
Yes but `copy` is not available on the `trait`. I have some API changes in the PR I opened to remove the second `::` in deps but I didn't want to move the discussion here.
For now we can live with this I guess and then refactor in case the other PR gets merged

Reply:
Oh right, sorry.